### ISSUE TITLE
Not tfrecord but npz

### ISSUE THREAD

**Issue by pzymua** [lightvector/KataGo#888]
When I ran shuffle.sh, the result was:
Number of rows by output file:
[('D:/KataGo/python/shuffleddata/20240120-093419/train\\data0.npz', 75008), ('D:/KataGo/python/shuffleddata/20240120-093419/train\\data1.npz', 74752), ('D:/KataGo/python/shuffleddata/20240120-093419/train\\data2.npz', 74752), ('D:/KataGo/python/shuffleddata/20240120-093419/train\\data3.npz', 74752)]
The output files are't tfrecord.
How could I resolve it?

**Comment by lightvector** [comment]
Tensorflow training is no longer implemented or supported. If you need to train with Tensorflow, you need to use a much older version of the repo. Pytorch training uses the npz files.

**Comment by pzystar** [comment]
Thanks.
But it seemed to be wrong. When I exported the weight model.bin.gz, the weight size was only 7KB(b5c192nbt) and then the load failed.

**Comment by pzystar** [comment]
[model.bin.gz](https://github.com/lightvector/KataGo/files/14065587/model.bin.gz)
![error](https://github.com/lightvector/KataGo/assets/66785026/792d8bae-b89e-4435-ac02-e97b35833bc8)
